### PR TITLE
Master

### PR DESCRIPTION
**Pull request by peachpellen**
first post

### CODE REVIEW

**Comment by aegaas** [comment]
@peachpellen hello, it looks like the PR build is failing on Travis. I have made some fixes on the master branch to stop that decryption command from running on Pull Requests. Would you please merge our master branch into this PR and that should allow the full build to run.

Thanks!

**Comment by peachpellen** [comment]
Hey - hoping this goes to your email though I'm about to check my work
inbox and you may be there too - I'm getting an error still, enoent - which
is absurd as I've got the whole repo 100% open right now, and I've only
seen this before with access rights being wrong. (This is npm test output,
below.)

I feel like I have to be missing something obvious but I've got no clue
what; I've never used a setup like this for blog, but I'm hoping you can
shed some light.

Thanks much


Peachs-MacBook-Pro:peachpellen peachpellen$ npm test

> intel-android-hub@0.0.1 test /Users/peachpellen/androidhub
> node jasmine-runner.js

Spec started

  Authors
    ✓ filename is a string

    data/authors/admin.yaml

      yaml data
        ✓ yaml parses into an object

        twitter key
          ✓ is required
          ✓ is not too long (15 characters)

    ✓ filename is a string

    data/authors/argyleink.yaml

      yaml data
        ✓ yaml parses into an object

        twitter key
          ✓ is required
          ✓ is not too long (15 characters)

    ✓ filename is a string

    data/authors/peachpellen.yaml

      yaml data
        ✓ yaml parses into an object

        twitter key
          ✓ is required
          ✓ is not too long (15 characters)

    ✓ filename is a string

    data/authors/TomDawsonAH.yaml

      yaml data
        ✓ yaml parses into an object

        twitter key
          ✓ is required
          ✓ is not too long (15 characters)

  Posts
    ✓ filename is a string
    ✓ contribute.jade read the file in as a string

    contribute.jade is a jade file with yaml frontmatter

      author
        ✓ is required
        ✓ is not too long (20 characters)

        has matching author.yaml
          ✓ file exists
          ✓ yaml parses into object

      yaml data
        ✓ yaml parses into object

        title field
          ✓ is required
          ✓ is not too long (200 characters)

        categories key
          ✓ is an array of strings
          ✓ at least 1 category
          ✓ but not too many (3 max)
          ✓ contains valid categories

      heroimage key
        ✓ is a url
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn identify ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
npm ERR! Test failed.  See above for more details.

**Comment by smhsieh** [comment]
@peachpellen got your message :) our team is looking into it and will follow up shortly!

File: posts/peachpellen/5_Apps_for_Android.jade [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title:      '5 Apps for Android Developers'
+categories: [ideate, launch, design, build]
+heroimage:  ''
```
Comment (aegaas) [inline posts/peachpellen/5_Apps_for_Android.jade]: Looks like you need to set a heroimage: please add an image to the library folder created for you and reference it here. The template will pick it up and use it in the header of your post and any listing pages.

Thanks!

Comment (smhsieh) [inline posts/peachpellen/5_Apps_for_Android.jade]: Hey Peach - can you also update your categories line (line 3)? We will want to classify your post under 1 of the 4 categories.

**Comment by argyleink** [comment]
helped ya out in this merge request: https://github.com/peachdev/androidhub/pull/1

**Comment by peachpellen** [comment]
Hey - I didn't merge the pull request as I am rusty with that, but I did
pull that code, modify it to add the hero image and then pushed it to the
pr/1 branch.

I did this with a new account on a new machine (I mean legitimately new, it
hasn't been used for dev at all before, so all clean installs etc) and
still, the same error.

That's also why I did a new fork, to create myself from scratch, all that -
but no luck.

I also tried running npm test on the fork prior to any changes from me
without success.

I understand that this is likely something stupid that I'm
missing/doing/etc. but I'm having a hard time seeing what as it's identical
in the new environment.

Sorry - I appreciate the help with modifying the file there. I'm going to
go and ask an engineer or two on my side and see if anyone knows what might
be up.

**Comment by argyleink** [comment]
Try running `npm install` again from your project directory, maybe some of the dependencies were missed in the first installation. It’s probably not something stupid you’re doing =) Stuff happens, we’ll help ya work through it. 

I started a gitter channel for us to talk, visit it here! https://gitter.im/intel-android/androidhub



On September 28, 2015 at 2:21:38 PM, Peach Pellen (notifications@github.com) wrote:

Hey - I didn't merge the pull request as I am rusty with that, but I did
pull that code, modify it to add the hero image and then pushed it to the
pr/1 branch.

I did this with a new account on a new machine (I mean legitimately new, it
hasn't been used for dev at all before, so all clean installs etc) and
still, the same error.

That's also why I did a new fork, to create myself from scratch, all that -
but no luck.

I also tried running npm test on the fork prior to any changes from me
without success.

I understand that this is likely something stupid that I'm
missing/doing/etc. but I'm having a hard time seeing what as it's identical
in the new environment.

Sorry - I appreciate the help with modifying the file there. I'm going to
go and ask an engineer or two on my side and see if anyone knows what might
be up.

**Comment by peachpellen** [comment]
I passed out on my face, sorry about that. I was up all night on very
little sleep writing Ruby. (Spoiler alert: I don't actually *know* Ruby.
Hence the fatigue.)

I'll run it again, and look at that "gitter" - I read "glitter" - I've not
used it before, but cool :-)

I will let you know the results.

**Comment by aegaas** [comment]
Looks like we have some normal test failures now. Looks like you need to set a heroimage and also not have all 4 categories in your post. Please choose 1.

**Comment by smhsieh** [comment]
Updated PR with Peach's Post: with https://github.com/intel-android/androidhub/commit/1029bc8e5bd120891a73c67e221a779c3b2de9f2

**Pull request closed without merging**